rule-examples/src/main/resources/com/solibri/smc/api/examples/beginner/UIComponentExample.properties

```diff
@@ -4,7 +4,7 @@ AUTHOR=Solibri, Inc.
 AUTHOR_TAG=SOL
 UID=EX5
 VERSION=1.1
-DATE=2020-05-29
+DATE=2023-01-12
```

Comment:
I can't answer that question :( I just got simple directions about this PR from Matti before he went on vacation:

"the api documentation has zip files that you just unzip."

So that's what I did. The date is also confusing me.